spec/integration/cli_view_spec.rb

```diff
@@ -24,5 +24,37 @@
       expect(File.exist?(File.join(output_dir, "libraries", "awesomelib.html"))).to be(true)
       expect(File.exist?(File.join(output_dir, "libraries", "newlib.html"))).to be(true)
     end
+
+    context "inqlude-all.json" do
+      it "generates view" do
+        output_dir = given_directory
+        result = run_command(args: ["view", "--output-dir=#{output_dir}"])
+
+        expect(File.exist?(File.join(output_dir, "inqlude-all.json"))).to be(true)
+      end
+
+      it "checks format" do
+        output_dir = given_directory
+        result = run_command(args: ["view", "--output-dir=#{output_dir}"])
+        data = File.read(File.join(output_dir, 'inqlude-all.json'))
+
+        expect {
+          parsedData = JSON.parse(data);
```

Comment:
You are not using the `parsedData` variable, so you could just write `JSON.parse(data)`.